home-exercises/src/test/java/LetterCounterTest.java

```diff
@@ -0,0 +1,19 @@
+import extrapractice.bta.LetterCounter;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.util.HashMap;
+
+public class LetterCounterTest {
+    @Test
+    void onlyLettersPresent() {
```

Comment:
Būtu forši, ja būtu tests, kurš pārbauda arī vai vispār ir saskaitīts pareizi. Šobrīd ja paņemtu tekstu "Šis ir mans kods" un rezultātā tiktu atgriezts [i -> 1, k -> 1], tad tests izietu, lai gan skaits neatbilstu.
Mans ieteikums - uztaisi mapToString metode publisku, un testā paņem to failu, saskaiti, pārtaisi par String ar to Tevis jau uztaisīto metodi un tad salīdzini ar testā definētu String mainīgo, kurā ir sagaidāmais rezultāts.